llvm/test/MC/AArch64/armv9.6a-occmo.s

```diff
@@ -0,0 +1,14 @@
+# NOTE: Assertions have been autogenerated by utils/update_mca_test_checks.py
+// RUN: llvm-mc -triple aarch64-none-linux-gnu -show-encoding -mattr=+occmo -mattr=+mte < %s | FileCheck %s
```

Comment:
I think these tests should also check instructions are rejected with `-mattr=+nooccmo`

Reply:
Done